code/game/objects/items/storage/firstaid.dm

```diff
@@ -550,7 +550,7 @@
 	STR.set_holdable(list(
 		/obj/item/organ,
 		/obj/item/bodypart,
-		/obj/item/reagent_containers/food/snacks/icecream
+		/obj/item/food
```

Comment:
Is this intentional? Going from ice cream to any food.

Reply:
No it's a botched find and replace